service-base/src/main/java/org/eclipse/hono/service/plan/TenantPlanValidator.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2017 Contributors to the Eclipse Foundation
```

Comment:
this file is new, right? so it should be 2019 only

Reply:
Done